test/integration/default/controls/packages.rb

```diff
@@ -3,8 +3,8 @@
 # Prepare platform "finger"
 platform_finger = system.platform[:finger].split('.').first.to_s
 
-control 'TEMPLATE package' do
-  title 'should be installed'
+control 'TEMPLATE.package.install' do
+  title 'required package should be installed'
```

Comment:
```suggestion
  title 'The required package(s) should be installed'
```

Reply:
Same comment as above for the capital `T` in the title.